docs/apache-airflow/cli-and-env-variables-ref.rst

```diff
@@ -39,6 +39,7 @@ contributed by the community providers:
 
 * Celery Executor and related CLI commands: :doc:`apache-airflow-providers-celery:cli-ref`
 * Kubernetes Executor and related CLI commands: :doc:`apache-airflow-providers-cncf-kubernetes:cli-ref`
+* Edge Executor and related CLI commands: :doc:`apache-airflow-providers-edge:cli-ref`
```

Comment:
Let's remove this for now, and do it separately as Airflow docs currently won't have reference to edge provider

Reply:
Removed it in https://github.com/apache/airflow/pull/42051/commits/f7d5d939e01bf1a264e345c36becff7105e41917 to verify that the test pass

Reply:
Follow-up created here: https://github.com/apache/airflow/pull/43136 -- details indescription